prime-dotnet-webapi/Models/DocumentAccessToken.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+    [Table("DocumentAccessToken")]
+    public sealed class DocumentAccessToken : BaseAuditable
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public Guid Id { get; set; }
+
+        public Guid DocumentGuid { get; set; }
+
+        [NotMapped]
+        public string DownloadUrl
+        {
+            get => $"{PrimeConstants.BACKEND_URL}/file-download/{Id}";
```

Comment:
Is there an alternative to using `PrimeConstants` for getting the API URL?

Reply:
From what i gathered from when i had to use it, it is just the way it is.

Reply:
hmm not that i know of